test_dimensional_dbt/models/dim_product_where_clause.sql

```diff
@@ -0,0 +1,15 @@
+WITH
+
+{{ dimensional_dbt.source_builder('_stripe_source','chiquita_product_id::NUMBER', 'stripe') }}
+,{{ dimensional_dbt.source_builder(['snapshots', 'bluth_wms_inventory'], "REPLACE(vendor_id,'chi-','')::NUMBER", 'warehouse', "source") }}
+,{{ dimensional_dbt.source_builder('DIMENSIONAL_DBT_SNAPSHOTS.CHIQUITA_INVOICES','id::NUMBER', 'vendor', "raw") }}
+
+{% call dimensional_dbt.column_selection(['stripe','warehouse','vendor'],6 ,["","warehouse_d.vendor_id = 'chi-500'",""]) %}
```

Comment:
I like all the different breaking patterns here ;)

Reply:
Will this work? or is this supposed to break?